compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -90,12 +89,31 @@ import qualified Text.PrettyPrint.ANSI.Leijen      as PP
 -- Commands
 --------------------------------------------------------------------------------
 
-cmdIde :: Mod CommandFields Command
+data CommandName = Build
+                 | Clean
+                 | Compile
+                 | DamlDoc
+                 | DocTest
+                 | Ide
+                 | Init
+                 | Inspect
+                 | InspectDar
+                 | License
+                 | Lint
+                 | MergeDars
+                 | Migrate
+                 | Package
+                 | Test
+                 | Visual
+                 deriving (Ord, Show, Eq)
+newtype Command = Command (CommandName, IO ())
+
+cmdIde :: Opt.Mod Opt.CommandFields Command
 cmdIde =
-    command "ide" $ info (helper <*> cmd) $
-       progDesc
+    Opt.command "ide" $ Opt.info (Opt.helper <*> cmd) $
```

Comment:
Not a huge fan of importing `optparse-applicative` qualified. I realize that it might be useful for someone new to Haskell but for someone that has a bit more experience it is imho annoying both for writing and for reading code as it adds unnecessary clutter.